c/bwcat.c

```diff
@@ -188,6 +229,8 @@ int main(int argc, char *argv[]){
 
   int j=0;
   for(j=0;j<chromList->nKeys;j++){//Iterate through each contig and access file.
+
+    if(ignore != NULL && check_ignore(ignore,ign_count,chromList->chrom[j])) continue;
```

Comment:
@drjsanger , other compiler errors.  Not sure setup.sh is working at all anymore for the bigwig stuff:

````
./bwcat.c:233:5: warning: passing argument 1 of 'check_ignore' from incompatible pointer type [enabled by default]
     if(ignore != NULL && check_ignore(ignore,ign_count,chromList->chrom[j])) continue;
     ^
./bwcat.c:134:5: note: expected 'const char **' but argument is of type 'char **'
 int check_ignore(const char **ignore_list, const int ign_size, const char *contig){
     ^
/tmp/cc5MuPLg.o: In function `main':
/tmp/downloads/PCAP-core/c/./bwcat.c:314: undefined reference to `bwClose'
/tmp/downloads/PCAP-core/c/./bwcat.c:317: undefined reference to `bwClose'
/tmp/downloads/PCAP-core/c/./bwcat.c:318: undefined reference to `bwCleanup'
/tmp/downloads/PCAP-core/c/./bwcat.c:224: undefined reference to `bwOpen'
/tmp/downloads/PCAP-core/c/./bwcat.c:297: undefined reference to `bwClose'
/tmp/downloads/PCAP-core/c/./bwcat.c:298: undefined reference to `bwCleanup'
/tmp/downloads/PCAP-core/c/./bwcat.c:237: undefined reference to `bwInit'
/tmp/downloads/PCAP-core/c/./bwcat.c:245: undefined reference to `bwOpen'
/tmp/downloads/PCAP-core/c/./bwcat.c:269: undefined reference to `bwGetOverlappingIntervals'
/tmp/downloads/PCAP-core/c/./bwcat.c:286: undefined reference to `bwAddIntervals'
/tmp/downloads/PCAP-core/c/./bwcat.c:292: undefined reference to `bwDestroyOverlappingIntervals'
/tmp/downloads/PCAP-core/c/./bwcat.c:293: undefined reference to `bwClose'
/tmp/downloads/PCAP-core/c/./bwcat.c:314: undefined reference to `bwClose'
/tmp/downloads/PCAP-core/c/./bwcat.c:315: undefined reference to `bwDestroyOverlappingIntervals'
/tmp/downloads/PCAP-core/c/./bwcat.c:314: undefined reference to `bwClose'
/tmp/downloads/PCAP-core/c/./bwcat.c:253: undefined reference to `bwCreateHdr'
/tmp/downloads/PCAP-core/c/./bwcat.c:257: undefined reference to `bwCreateChromList'
/tmp/downloads/PCAP-core/c/./bwcat.c:262: undefined reference to `bwWriteHdr'
collect2: error: ld returned 1 exit status
make: *** [../bin/bwcat] Error 1
make: Leaving directory `/tmp/downloads/PCAP-core/c'
````